r4/fhirpath/Ballerina.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+org = "ballerinax"
+name = "health.fhir.templates.r4.fhirpath_api"
+displayName = "FHIRPATH API"
+description = "Exposes FHIRPATH Processing Engine as managed APIs"
+authors = ["Ballerina"]
+keywords = ["Healthcare", "FHIR", "FHIRPATH"]
+version = "0.1.0"
+distribution = "2201.5.0"
```

Comment:
Bal version

Reply:
Fixed in 28ab50418e064931acc8a5e17b22cbfa773c827a